packages/fxa-settings/src/components/PageTwoStepAuthentication/index.tsx

```diff
@@ -144,14 +165,22 @@ export const PageTwoStepAuthentication = (_: RouteComponentProps) => {
 
   const [verifyTotp] = useMutation(VERIFY_TOTP_MUTATION, {
     onCompleted: () => {
-      alertTextExternal('Two-step authentication enabled');
+      alertTextExternal(
+        l10n.getString('tfa-enabled', null, 'Two-step authentication enabled')
+      );
       goHome();
     },
     onError: (err) => {
       if (err.graphQLErrors?.length) {
         setRecoveryCodeError(err.message);
```

Comment:
You should be able to set localized error

```
const localizedError = l10n.getString(`auth-error-${error.errno}`);
```

Although, now that I take another maybe we can have graphql set this.

Reply:
Oops missed a spot.  Good catch.  Also, I don't know if gql-api gives us the errno from the original error; I'll double check.